artifactory/cli.go

```diff
@@ -2343,6 +2353,17 @@ func transferConfigCmd(c *cli.Context) error {
 	return nil
 }
 
+// Check if there is a configured user using default credentials 'admin:password' by pinging Artifactory.
+func isDefaultCredentials(url string) bool {
+	return pingWithDefaultCred(url) == nil
+}
+
+func pingWithDefaultCred(url string) error {
+	artDetails := coreConfig.ServerDetails{ArtifactoryUrl: url, User: "admin", Password: "password"}
+	pingCmd := generic.NewPingCommand().SetServerDetails(&artDetails)
+	return commands.Exec(pingCmd)
```

Comment:
Let's reunite the functions into 1
```suggestion
	artDetails := coreConfig.ServerDetails{ArtifactoryUrl: url, User: "admin", Password: "password"}
	pingCmd := generic.NewPingCommand().SetServerDetails(&artDetails)
	return commands.Exec(pingCmd) == nil
```